notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb

```diff
@@ -276,7 +346,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ResponsibleAIDashboard(rai_insights)"
+    "widget = ResponsibleAIDashboard(rai_insights, cohort_list=cohort_list)"
```

Comment:
why does "widget =" need to be assigned here?

Reply:
Removed it.